### PR TITLE
173 create geteventvolunteersbyorganization server function

### PR DESCRIPTION
**Pull request by Frank-D-stein**
# Description
returns all event volunteer responses who have attended an event by an organization

## Relevant issue(s)
- #173 

## Questions
- [x] Was I supposed to use EventVolunteerResponse or VolunterEventResponse? 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have assigned reviewers to this PR

# Test code
```
import { getEventVolunteersByOrganization } from '@/server/actions/Volunteer';
import { EventVolunteerResponse } from '@/types/dataModel/eventVolunteer';

export default async function Home() {
  const eventVolunteers = await getEventVolunteersByOrganization(
    '65da9486c8488f16e6bf6116'
  );

  return (
    <div>
      <h1>Home page</h1>
      <h2>Event Volunteers:</h2>
      <ul>
        {eventVolunteers.map((volunteer: EventVolunteerResponse) => (
          <li key={volunteer._id}>
            <p>
              Name: {volunteer.volunteer.firstName}{' '}
              {volunteer.volunteer.lastName}
            </p>
          </li>
        ))}
      </ul>
    </div>
  );
}
```

### CODE REVIEW

File: src/server/actions/Volunteer.ts
Comment (andlrutt) [inline src/server/actions/Volunteer.ts:1]: Just as is done on line 20 of this file with `OrganizationSchema`, import and declare `EventSchema`. If the server hasn't hit the events collection directly before in its current runtime, the schema will not be defined and mongoose will freak out. Importing and declaring it ensures that the code that defines the schema is run

**Review by andlrutt — CHANGES_REQUESTED** [review]
It's functional, but change all occurrences of `EventVolunteer` in comments, var names, etc. to `volunteerEvent`

**Comment by andlrutt** [comment]
> It's functional, but change all occurrences of `EventVolunteer` in comments, var names, etc. to `volunteerEvent`

Also fix merge conflicts :)

File: src/server/actions/Volunteer.ts
Comment (Frank-D-stein) [reply src/server/actions/Volunteer.ts:1]: Ahh that was what was causing the issue. I was working on fixing that but I ran out of time. Thank you

**Review by andlrutt — APPROVED** [review]
Good stuff!

**Pull request merged**